src/store/middlewares/roomMiddleware.tsx

```diff
@@ -108,6 +138,61 @@ const createRoomMiddleware = ({
 				});
 			}
 
+			if (
+				getState().settings.notificationSounds &&
+				notificationsActions.enqueueNotification.match(action)
+			) {
+				signalingService.on('notification', (notification) => {
+					try {
+						switch (notification.method) {
+							case 'raisedHand':
+							case 'chatMessage':
+							case 'parkedPeer':
+							case 'parkedPeers':
+							case 'sendFile':
+							case 'newPeer': {
+								if (action.payload.playSound) {
+									const type = notification.method;
+
+									const soundAlert = soundAlerts[type] === undefined ?
```

Comment:
Simplify this to: `const soundAlert = soundAlerts[type] ?? soundAlerts['default']`

Reply:
Done